web/collections/_documentation/validation-schemas.md

```diff
@@ -115,3 +115,18 @@ schemas:
     path: "schemas/bar.json"
     title: "Une description de ce sous-schéma"
 ```
+
+## Validations spécifiques au format générique
```

Comment:
Explain why this format is needed / useful.

Too bad we're introducing this because I think we've got guidance about _why_ a formal schema, and not words only, is useful 😬